.circleci/config.yml

```diff
@@ -88,6 +88,8 @@ jobs:
     parameters:
       env:
         type: string
+      env_type:
```

Comment:
Thought I'd make this a new var explicitly so we have the option of adding `regression` later and are less likely to accidentally pass it as Jira environment_type which I think wouldn't work.